### PR TITLE
Add async as option for forcesave command

### PR DESCRIPTION
**Pull request by DoctorFTB**
PR for https://github.com/ONLYOFFICE/DocumentServer/issues/1307

### CODE REVIEW

**Comment by ShockwaveNN** [comment]
@DoctorFTB Could you change destanation branch to `develop` - we only support 3rd party request to develop

**Comment by DoctorFTB** [comment]
> @DoctorFTB Could you change destanation branch to `develop` - we only support 3rd party request to develop

Changed.

File: DocService/sources/DocsCoServer.js [hunk 1]
```diff
@@ -3289,7 +3289,9 @@ exports.commandFromServer = function (req, res) {
             }
             break;
           case 'forcesave':
-            let forceSaveRes = yield startForceSave(docId, commonDefines.c_oAscForceSaveTypes.Command, params.userdata, undefined, undefined, undefined, undefined, utils.getBaseUrlByRequest(req));
+            // If async was not send - set true (as default)
+            let async = (typeof params.async === 'undefined') ? true : (typeof params.async === 'string') ? 'true' === params.async : params.async;
```
Comment (konovalovsergey) [inline DocService/sources/DocsCoServer.js:3293]: by analogy with https://api.onlyoffice.com/editors/conversionapi#async, `async ` should be boolean only

Comment (DoctorFTB) [reply DocService/sources/DocsCoServer.js:3293]: This ternary based on this code for async variable:
https://github.com/ONLYOFFICE/server/blob/master/DocService/sources/converterservice.js#L266
https://github.com/ONLYOFFICE/server/blob/master/DocService/sources/converterservice.js#L328

Comment (konovalovsergey) [reply DocService/sources/DocsCoServer.js:3293]: unfortunately this PR is not enough. parameter does not work. sync cycle monitors another key

Comment (DoctorFTB) [reply DocService/sources/DocsCoServer.js:3293]: Hmm. I see variable `docId` which used in cycle `selectRes = yield taskResult.select(docId);` (136 line) is correct. Its same key which sended in command.

Comment (DoctorFTB) [reply DocService/sources/DocsCoServer.js:3293]: Any news?

**Comment by DoctorFTB** [comment]
Thank you very very very much for helping :thumbsup:

**Pull request closed without merging**